### PR TITLE
Choose friend page

### PR DESCRIPTION
**Pull request by bun919tw**
* One activity contains a viewpager and a tab layout
* Viewpager contains three fragments

### CODE REVIEW

**Comment by bun919tw** [comment]
附上兩張截圖
screenshot_20160525-003246
screenshot_20160525-003254

File: PointTreeUser/app/src/main/java/tw/com/pointtree/pointtreeuser/activities/UserQueryActivity.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            }
+
+            @Override
+            public void onTabReselected(TabLayout.Tab tab) {
+                // remain empty
+            }
+        });
+    }
+
+    private static class StartPagerAdapter extends FragmentPagerAdapter {
```
Comment (StanleyDing) [inline PointTreeUser/app/src/main/java/tw/com/pointtree/pointtreeuser/activities/UserQueryActivity.java]: `Start`PagerAdapter?

**Comment by StanleyDing** [comment]
圖片只有一種解析度

**Comment by bun919tw** [comment]
@StanleyDing 改好了

**Pull request merged**